### PR TITLE
Add pages company aa1 to implement 1

### PR DESCRIPTION
**Pull request by pinterid**
The companypage component has been
added. The page is not routed yet.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no companypage. #37 

**What is the new behavior (if this is a feature change)?**
- Now there is a companypage. #37 

**Other information**:
- Ref: 🐍

### CODE REVIEW

File: src/components/pages/CompanyPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+// Configure tabs
+const TABS = [
+  "Overview",
+  "People",
+  "Talks",
+  "Locations",
+  "Platforms",
+  "Milestones",
+  "About",
+];
```
Comment (schettn) [inline src/components/pages/CompanyPage/index.jsx:45]:
```suggestion
const TABS = [
  "Overview",
  "People",
  "Talks",
  "Locations",
  "Platforms",
  "Milestones",
  "About",
];

```

**Review by schettn — CHANGES_REQUESTED** [review]
I left some suggestions.

**Review by schettn — APPROVED** [review]
LGTM

File: src/components/pages/CompanyPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+
+//> CSS
+import "./company.scss";
+//> Data
+// Dummy data
+import data from "./dummy/data.js";
+import platformData from "./dummy/chart.json";
+//#endregion
+
+//#region > Static Variables
```
Comment (pinterid) [inline src/components/pages/CompanyPage/index.jsx]:
```suggestion
//#region > Constant Variables
```

File: src/components/pages/CompanyPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+const TABS = [
+  "Overview",
+  "People",
+  "Talks",
+  "Locations",
+  "Platforms",
+  "Milestones",
+  "About",
+];
+
```
Comment (pinterid) [inline src/components/pages/CompanyPage/index.jsx:46]:
```suggestion
```

**Review by pinterid — COMMENTED** [review]
Some changes have to be made.

File: src/components/pages/CompanyPage/index.jsx [hunk 4]
```diff
@@ -0,0 +1,676 @@
...
+                            <MDBCol md="4" key={i}>
+                              <MDBCard>
+                                <MDBCardBody>
+                                  <p className="font-weight-bold mb-0 d-inline-block clickable blue-text">
+                                    {employee.full_name}
+                                  </p>
+                                  <p className="text-muted">
+                                    {employee.position}
+                                  </p>
+                                  {employee.joined && (
```
Comment (kleberbaum) [inline src/components/pages/CompanyPage/index.jsx:482]: Please fix indentation.
```suggestion
                                    {employee.joined && (
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**